packages/strapi-admin/admin/src/components/EmptyList/Wrapper.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ * Wrapper
+ *
+ */
+
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  padding: 54px 30px 30px;
+  text-align: center;
+  background-color: white;
+  p,
```

Comment:
I won't create a component just to set a `line-height: normal` attribute. I wish it's a problem we will never have again after setting the new design system: this attribute should be set to all elements in a reset file, it's the way the entire admin should work.

For the `<p/>` element, the selector `:first-of-type` makes it non global.

Reply:
I see, you are right. Regarding the reset css do you thing that this is a task we should undertake soon?
I know that we have an import of [sanitize.css](https://csstools.github.io/sanitize.css/) in the entry point of the admin do you think that we should remove it?

Reply:
It's a task we should indeed undertake soon, but it may take some time to check it doesn't broke the grid :( 
`sanitize.css` is great for some generic attributes but it seems we cannot custom it. Maybe we can replace it by our own reset rules (like `button` hasn't an `outline`, `p` doesn't have a `margin-bottom` by default, or all the `ul` has a list-style` attribute set to `none`, etc) based on the future design system so we're sure not to specify it each time.